go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/distros-test-framework
 
-go 1.20
+go 1.21
+
+toolchain go1.21.8
```

Comment:
I think this needs a rebase onto main.. this should already be present in go.mod there